test/mocks/fixtures/generate-download-url.js

```diff
@@ -4,7 +4,7 @@ const generateDownloadUrl = {
   method: 'POST',
   response: {
     status: 201,
-    body: 'http://www.google.com',
+    body: 'https://en.wikipedia.org/static/images/project-logos/enwiki.png',
```

Comment:
I'm not sure we want to commit this?

Reply:
it was intentional, just to have an image for the tests